docs/integrations/pytorch.rst

```diff
@@ -23,8 +23,15 @@ it takes just one line of Python to convert a HuggingFace dataset to a Lance dat
     lance.write_dataset(hf_ds, "diffusiondb_train.lance")
 
 Then, you can use the Lance dataset in PyTorch training and inference loops.
-Not that the PyTorch dataset automatically convert data into :class:`torch.Tensor`
 
+Note:
+
+1. the PyTorch dataset automatically convert data into :class:`torch.Tensor`
+
+2. lance is not fork-safe. If you are using multiprocessing, use spawn instead.
```

Comment:
Maybe we can add a note here, e.g. `The community has included it in the 2025 Roadmap and will actively address this issue.` @eddyxu WDYT?